src/components/icons/index.ts

```diff
@@ -0,0 +1,15 @@
+import { MagnifierIcon } from "./MagnifierIcon";
+import { SortVerticalIcon } from "./SortVerticalIcon";
+import { HeartIcon } from "./HeartIcon";
+import { CloseIcon } from "./CloseIcon";
+import { ArrowIcon } from "./ArrowIcon";
+import { SidebarIcon } from "./SidebarIcon";
+
+export {
+  MagnifierIcon,
+  SortVerticalIcon,
+  HeartIcon,
+  CloseIcon,
+  ArrowIcon,
+  SidebarIcon,
+};
```

Comment:
instead of importing and then exporting you can:
`export * from "./MagnifierIcon";
export * from "./SortVerticalIcon";
export * from "./HeartIcon";
export * from "./CloseIcon";
export * from "./ArrowIcon";
export * from "./SidebarIcon";`